front-end/src/pages/Error.jsx

```diff
@@ -1,13 +1,115 @@
 import './Error.css';
 
-import React from 'react';
+import React, { useEffect } from 'react';
+import Button from "../components/Button";
+import { Link } from "react-router-dom";
+
+import Cookies from 'universal-cookie';
+const cookies = new Cookies();
+
+function Error(props) {
+  const [error, setError] = React.useState("undefined");
+  const [group, setGroup] = React.useState("undefined");
+  const [user, setUser] = React.useState("undefined");
+  const [next, setNext] = React.useState("/");
+
+  const resetCookies = () => {
+    cookies.remove("groupID");
+    cookies.remove("groupName");
+    cookies.remove("user");
+    window.location.href = `${next}`;
+    // cookies.remove("cuisine");
+    // cookies.remove("preferred");
+  };
+
+  useEffect(() => {
+    const nextLink = props.location.state.next ? props.location.state.next : "/";
+    setNext(nextLink);
+    setError(props.location.state.error);
+    setGroup(props.location.state.group);
+    setUser(props.location.state.user);
```

Comment:
```suggestion
    const nextLink = props.location.state?.next ? props.location.state?.next : "/";
    setNext(nextLink);
    setError(props.location.state?.error);
    setGroup(props.location.state?.group);
    setUser(props.location.state?.user);
```
Dunno if this is the best way to fix it, but it seems to at least fix the crash in #143